thirdparty/harfbuzz/CMakeLists.txt

```diff
@@ -60,8 +60,6 @@ ExternalProject_Add(
     BUILD_IN_SOURCE 1
     PATCH_COMMAND COMMAND ${PATCH_CMD} COMMAND ${AUTOGEN_CMD}
     CONFIGURE_COMMAND ${CFG_CMD}
-    # Normally we would use -j${PARALLEL_JOBS}, which is ${PROCESSOR_COUNT} + 1.
-    # But on CircleCI that means 3 instead of 2 and a risk of OOM.
-    BUILD_COMMAND ${KO_MAKE_RECURSIVE} -j${PROCESSOR_COUNT}
+    BUILD_COMMAND ${KO_MAKE_RECURSIVE} -j${CONSTRAINED_PROCESSOR_COUNT}
```

Comment:
It just occurred to me that perhaps it should be CONSTRAINED_PARALLEL_JOBS, so it's still greppable.

Reply:
Yup, I had a vaguely similar thought ;).